guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>X-Conv is recursively applied to<s> “project”, or “aggregate”</s>, <strong>“project” or “aggregate”</strong> information <s>from neighborhoods</s> <strong>from the neighborhood</strong></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
''corrected accordingly'', Thanks.